src/mcts/node.h

```diff
@@ -253,6 +253,11 @@ class Node {
   friend class Edge;
 };
 
+// Define __i386__ also for 32 bit Windows.
+#if defined(_WIN32) && !defined(_WIN64)
+#define __i386__ 1
```

Comment:
Why `1`? Just `#define __i386__` should work.

Reply:
No reason. However this is not doing the right thing for 32 bit arm processors, so I need to revise it.

Reply:
Is there windows on arm?